lib/modules/dosomething/dosomething_reportback/dosomething_reportback.module

```diff
@@ -891,6 +891,7 @@ function dosomething_reportback_get_reportback_total_by_nid($nid, $type = 'quant
 
   // Select all reportbacks for given $nid.
   $query = db_select('dosomething_reportback', 'rb')->condition('nid', $nid);
+  $query->condition('promoted', 0); // Prevents non-reviewed RB's from being added to total
```

Comment:
if we search by null then you get reportbacks inside the inbox. thats why I did 0 (and now suggesting 0 or 1) because only reportbacks not in the inbox have one of those values